src/Resources/app/administration/src/module/swag-migration/store/migration.store.ts

```diff
@@ -364,8 +364,22 @@ const migrationStore = Shopware.Store.register({
             try {
                 this.environmentInformation = await migrationApiService.checkConnection(this.connectionId);
                 this.lastConnectionCheck = new Date();
-            } catch {
-                await this.createErrorNotification('swag-migration.api-error.checkConnection');
+            } catch (error) {
+                const code = error?.response?.data?.errors[0]?.code;
+
+                if (!code) {
+                    await this.createErrorNotification('swag-migration.api-error.checkConnection');
+                    return;
+                }
+
+                const errorMessageSnippet = `swag-migration.wizard.pages.credentials.error.${code}`;
+
+                if (!Shopware.Snippet.tc(errorMessageSnippet)) {
```

Comment:
This if branch will never be executed (it should always return a string, even if the it's just the key if it doesn't exists). I guess you want to check if the snippet exists?

```suggestion
                if (!Shopware.Snippet.te(errorMessageSnippet)) {
```